src/applications/disability-benefits/all-claims/tests/pages/toxicExposure/gulfWar2001Details.unit.spec.jsx

```diff
@@ -85,11 +85,14 @@ describe('gulfWar2001Details', () => {
         }
       });
 
-      it(`should submit without dates for ${locationId}`, () => {
+      // TODO: We currently validate against this on the frontend to prevent the 'XX' date issue,
+      // however we want Veterans to be able to submit with a completely blank date.
+      // Note to revisit after we land on a solution for accommodating partial dates.
+      it(`should not submit without dates for ${locationId}`, () => {
         pageSubmitTest(
           schemas[`gulf-war-2001-location-${locationId}`],
           formData,
-          true,
+          false,
         );
       });
 
```

Comment:
Would be beneficial to include in this spec tests that verify that submission should not occur when:
- the start and/or end dates are incomplete,
- when an end date is before a start date, and
- when the end date is before September 11, 2001